src/cljs/org/broadinstitute/firecloud_ui/common/components.cljs

```diff
@@ -80,4 +80,51 @@
                               (when-let [f (:onTabSelected tab)] (f e)))}])
            (:items props))
          [:div {:style {:clear "both"}}]]
-        [:div {} (:component (nth (:items props) (:active-tab @state)))]])}))
+        [:div {} (:component (nth (:items props) (or (:active-tab @state) 0)))]])}))
```

Comment:
Ah, the "or 0" thing was because the state was getting screwed up on hot reloading.  I'll take it out.